benchmarks/scala-dotty/src/main/scala/org/renaissance/scala/dotty/Dotty.scala

```diff
@@ -80,12 +80,34 @@ class Dotty extends RenaissanceBenchmark {
   private val DOTTY_ARG_TYPE_CONVERSION = "-language:implicitConversions"
 
   override def runIteration(c: Config): BenchmarkResult = {
+    /*
+     * Construct the classpath for the compiler. Unfortunately, Dotty is
+     * unable to use current classloader (either of this class or this
+     * thread) and thus we have to explicitly pass it. Note that
+     * -usejavacp would not work here as that reads from java.class.path
+     * property and we do not want to modify global properties here.
+     *
+     * Therefore, we leverage the fact that we know that our classloader
+     * is actually a URLClassLoader that loads the benchmark JARs
+     * from temporary directory. And we convert all the URLs to
+     * plain file paths.
+     *
+     * Note that using URLs as-is cannot be used as that prepends the
```

Comment:
"Note that using URLs as-is is not possible ..."

Reply:
Thanks!